src/helpers.rs

```diff
@@ -304,4 +304,11 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
     fn eval_libc_i32(&mut self, name: &str) -> InterpResult<'tcx, i32> {
         self.eval_libc(name)?.to_i32()
     }
+
+    fn avoid_isolation(&mut self, name: &str) -> InterpResult<'tcx> {
```

Comment:
"avoid" is somewhat weak. Maybe `check_no_isolation`?

Reply:
I'm renaming this to `banish_isolation` :P